users/user-guide-query/multi-stage-query/operator-types/union.md

```diff
@@ -0,0 +1,113 @@
+---
+description: >-
+  Describes the union relation operator in the multi-stage query engine.
+---
+
+# Union relational operator
+
+The union operator combines the results of two or more queries into a single result set.
+The result set contains all the rows from the queries.
+Contrary to other set operations ([intersect](intersect.md) and [minus](minus.md)), the union operator does not remove
```

Comment:
this confused me a bit.. although the operator doesnt remove duplicates, we have a way of achieving union by adding the aggregate. So from end user perspective the functionality works. Reading this made me first think we don't support the functionality

Reply:
I don't get what do you mean. Are you suggesting that the user can add a `distinct` or an `aggregate` phase after the union so they can achieve the same thing?

Reply:
Oh, I guess you mean that we actually honor the fact that `all` is provided or not. Yes, the text is pretty confusing.